Tests/Development.server.lua

```diff
@@ -7,7 +7,25 @@ MetrikSDK.Server:SetProjectId(ServerStorage.__Project_Id.Value)
 MetrikSDK.Server:SetAuthenticationToken(ServerStorage.__Project_Auth.Value)
 
 MetrikSDK.Server:InitializeAsync():andThen(function()
-	-- warn("Metrik SDK loaded!")
+	warn("Metrik SDK loaded!")
+	
+	local action = MetrikSDK.Server.ActionBuilder.new()
+		:SetName("Hello, world!")
+		:SetDescription("A simple 'Hello, World' print!")
+		:AddArgument("String", {
```

Comment:
What about a shorthand/bulk argument 'adder' as `:AddArguments`?

Reply:
Probably worth keeping it simple atm, then scaling it up once we know things are up n working :smile:

Reply:
sure